shelter.h

```diff
@@ -3,6 +3,7 @@
 
 #include "color.h"
 #include <cmath>
+#include "string.h"
```

Comment:
The inclusion of an STL header file must be with with angle brackets [SF.12: Prefer the quoted form of #include for files relative to the including file and the angle bracket form everywhere else](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-incform). 

Either fix, or I will create an Issue to let a beginner fix this :+1: